CONTRIBUTING.md

```diff
@@ -67,7 +67,7 @@ And re-run `pnpm install` to link the package.
 
 Each package under `packages/playground/` contains a `__tests__` directory. The tests are run using [Jest](https://jestjs.io/) + [Playwright](https://playwright.dev/) with custom integrations to make writing tests simple. The detailed setup is inside `jest.config.js` and `scripts/jest*` files.
 
-Each test can be run under either dev server mode or build mode.
+Each test can be run under either dev server mode or build mode. Make sure that [Vite has been built](#repo-setup).
```

Comment:
I think we should note that all the packages besides Vite (e.g. plugin-vue) should be built too though. I believe the tests are using all the packages, so noting we have to run `pnpm build` in the root directory initially may be useful.